### PR TITLE
implement npm workspaces

### PR DESCRIPTION
**Pull request by DeityLamb**
I added an npm registry to the .npmrc file to simplify local testing, hopefully this won't be a problem

### CODE REVIEW

**Comment by OleksandrKucherenko** [comment]
image

I got such error when try to apply those changes

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "diia-inhouse",
   "scripts": {
-    "build": "npm run build --ws"
+    "build": "npm run build --ws --if-present"
```
Comment (DeityLamb) [inline package.json:4]: It seems that the `eslint-config` package does not have a build script. Similar cases will be ignored with the `--if-present` flag,  assuming that this is the behavior we desire

File: .npmrc [hunk 2]
```diff
@@ -1,3 +1,4 @@
 # https://verdaccio.org/docs/authentication/
 
 registry=https://registry.dev:4873
+registry=https://registry.npmjs.org
```
Comment (OleksandrKucherenko) [inline .npmrc:4]: ref: https://stackoverflow.com/questions/32633678/is-there-any-way-to-configure-multiple-registries-in-a-single-npmrc-file

i think we should use local registry only for @diia-inhouse scope

**Comment by OleksandrKucherenko** [comment]
Replaced by latest commits.

**Pull request closed without merging**